packages/graphql-transformer-core/src/GraphQLTransform.ts

```diff
@@ -331,6 +332,20 @@ export class GraphQLTransform {
     context: TransformerContext,
   ) {
     let index = 0;
+
+    // hashing key directive to get the order in @key transformer
+    let ind = 0;
```

Comment:
index, ind, can we come up with a better naming and less abbreviations? like "dir" down below, that increases the code readibility a lot.

Reply:
done